docker/elasticsearch/elasticsearch.yml

```diff
@@ -26,4 +30,4 @@ xpack.graph.enabled: false
 xpack.watcher.enabled: false
 
 #Accept all requests from localhost, dashboard and kibana
-#network.host: [127.0.0.1, localhost, dashboard, kibana, nginx]
```

Comment:
@fxprunayre the requests from kibana and dashboard are coming from `kibana` and `dashboard`, **not** from `localhost`.